### PR TITLE
Client v2

### PR DESCRIPTION
**Pull request by rsarv3006**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "apialerts"
-version = "0.1.0"
+version = "2.0.0"
```
Comment (mononz) [inline Cargo.toml:3]: Lets jump to 1.0.0 rather than 2. I'll push a few other standardising updates along with the 2.0 release for all sdks

Comment (rsarv3006) [reply Cargo.toml:3]: 🤦  doh I forgot

Comment (rsarv3006) [reply Cargo.toml]: done

**Review by mononz — APPROVED** [review]
:shipit:

**Pull request merged**